funnel/views/project.py

```diff
@@ -572,6 +573,9 @@ def register(self):
                 rsvp.rsvp_yes()
                 db.session.commit()
                 flash(_("You have successfully registered"), 'success')
+                send_email_for_project_registration(
```

Comment:
Send the signal here. Don't call the email function directly. The signal's sender is the `rsvp` object that was just created.

Reply:
I made changes. Does [this](https://github.com/hasgeek/funnel/blob/5fdc63d90cf3a9cf5d28664ca14058a483368370/funnel/views/project.py) look OK?